templates/nginx/all-services-config.yaml

```
@@ -0,0 +1,39 @@
+{{- if .Values.nginx.enabled }}
```

In my env I would not want these admin pages accessible to most users. The simplest way to achieve this would be to just disable the service index. Can you make it optional?
```suggestion
{{- if and .Values.nginx.enabled .Values.nginx.serviceIndex.enabled }}
```

---

hmm I can absolutely do that (not fully done anyway) but I don't know really know what qualifies as an admin page. I really wanted to do everything (including supporting delete requests) and then let the user decide what is best for his corp. So either

- Use at is it
- Add Authentication & Authorization infront of it (Should be done anyway)
- Completely roll your own

---

Yes, I would leave authorization up to the user. My requests here was to provide a mechanism to disable this feature entirely, like how the chart is today. Since it's not required to run Cortex it can be an optional feature of the chart.

---

I mean yeah sure. If we could maybe define what would be considered an "admin endpoint" then we could make these optional for the user?